### ISSUE TITLE
Podcasts disappear

### ISSUE THREAD

**Issue by BPDMF** [podStation/podStation#243]
Just as the title says. I import podcasts from opml or search for them and add them but all of them are gone the next time I open up podstation. Am I the only one? This can't be expected behavior, right?

**Comment by dellagustin** [comment]
Hello @BPDMF, that is for sure not the expected behavior, they are supposed to be persisted.
No one ever reported this issue.
Based on your comment, I assume that was the behavior since your first usage, right? It never worked in a different way for you?

I'll give it some thought on how we could investigate this issue, thanks for reporting.

In the meantime, are you using it on Chrome or another browser? Which version?

**Comment by BPDMF** [comment]
The podcasts disappearing has happened since i first used it. I've kept
using it for about a year now just using an opml file on my desktop to
regenerate the podcasts when i want to listen, it's a hassle but it's
worked. I expected a fix at some point but i guess if i never bug report
you'll never know to fix it so i finally decided to. Using chrome on
windows 10 and across maybe 4 different computers between desktops and
laptops and chrome is currently and usually up to date, same for windows,
and chrome is the standard version and not canary or whatever. FYI I've
only ever imported from opml, I've never once added from rss or any other
way

**Comment by dellagustin** [comment]
Hmm the fact that you are using it in 4 different computers and the issue happens in all of them is rather puzzling.
Are you logged in the browser with a google account?
This sounds like an issue on the sync storage, but it is hard to tell.
I tried to reproduce it here, but without success, importing an OPML in a new user profile, but it worked fine for me.
Can you inspect the console on the background page and send me a screenshot if there are any errors?
![image](https://user-images.githubusercontent.com/260113/117066797-7e59a280-ad29-11eb-9dc9-854e657ed4f8.png)

**Comment by BPDMF** [comment]
Yes, in using a Google account on all the instances of chrome. I'll do the
background thing or whatever but what exactly do i need to do? It looks
like dev mode on the chrome extensions page or something. Not sure exactly
what you want me to do.

**Comment by dellagustin** [comment]
Ok, so let me give you some more detailed steps:

1. Access this url: chrome://extensions/
2. Turn on the developer mode:
    ![image](https://user-images.githubusercontent.com/260113/117360821-86e0e300-aeb9-11eb-93b4-d873d649ba44.png)
3. Click on the extension details:
    ![image](https://user-images.githubusercontent.com/260113/117361106-e3dc9900-aeb9-11eb-898c-f10c9fd2d911.png)
4. Click on the background page
    ![image](https://user-images.githubusercontent.com/260113/117361230-0cfd2980-aeba-11eb-9cbe-df4117aa7bfd.png)
5. The DevTools will open, click on the Console, tab, scroll to see if there are any errors, send me a screenshot of the errors:
    ![image](https://user-images.githubusercontent.com/260113/117361375-3fa72200-aeba-11eb-9bbd-0fa80f1c0b57.png)

**Comment by dellagustin** [comment]
Hello, @BPDMF, just checking, did anything change for you in this one?

**Comment by BPDMF** [comment]
No. I just never got around to taking the steps in your last reply. I'll
get around to it over the next few days.
